src/Tizen.Multimedia.Vision/MediaVision/RoiTrackingConfiguration.cs

```diff
@@ -0,0 +1,175 @@
+ */
+
+using System;
+using InteropRoi = Interop.MediaVision.RoiTracker;
+
+namespace Tizen.Multimedia.Vision
+{
+    /// <summary>
+    /// Represents a configuration of <see cref="RoiTracker"/> instances.
+    /// </summary>
+    /// <feature>http://tizen.org/feature/vision.roi_tracking</feature>
+    /// <exception cref="NotSupportedException">The required features are not supported.</exception>
+    /// <since_tizen> 10 </since_tizen>
+    public class RoiTrackingConfiguration : EngineConfiguration
+    {
+        private IntPtr _handle;
+        private bool _disposed;
+        private const string KeyTrackerType = "MV_ROI_TRACKER_TYPE";
+
+        internal bool IsApplied { get; set; }
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="RoiTrackingConfiguration"/> class.
+        /// </summary>
+        /// <rema
+        /// <exception cref="NotSupportedException">The feature is not supported.</exception>
+        /// <since_tizen> 10 </since_tizen>
+        public RoiTrackingConfiguration() : base("roi_tracking")
+        {
+            InteropRoi.Create(out _handle).Validate("Failed to create roi tracking configuration");
+        }
+
+        internal IntPtr GetRoiConfigHandle()
+        {
+            return _handle;
+        }
+
+        /// <summary>
+        /// Gets or sets the type of ROI tracker.<br/>
+        /// Default value is <see cref="RoiTrackerType.Balance"/>.
+        /// </summary>
+        /// <exception cref="ArgumentException"><paramref name="value"/> is not valid.</exception>
+        /// <since_tizen> 10 </since_tizen>
+        public RoiTrackerType TrackerType
+        {
+            get
+            {
+                return (RoiTrackerType)GetInt(KeyTrackerType);
+            }
+            set
+            {
+                ValidationUtil.ValidateEnum(typeof(RoiTrackerType), value, nameof(value));
+
+                Set(KeyTrackerType, (int)value);
+
+                InteropRoi.Configure(_handle, GetHandle(this)).Validate("Failed to configure model type");
+            }
+        }
+
+        private Rectangle? _roi;
+
+        /// <summary>
+        /// Gets or sets the ROI(Region Of Interest) of <see cref="RoiTracker"/>.
+        /// </summary>
+        /// <exception cref="ArgumentOutOfRangeException">
+        ///     The width of <paramref name="value"/> is less than or equal to zero.<br/>
+        ///     -or-<br/>
+        ///     The height of <paramref name="value"/> is less than or equal to zero.<br/>
+        ///     -or-<br/>
+        ///     The x position of <paramref name="value"/> is less than zero.<br/>
+        ///     -or-<br/>
+        ///     The y position of <paramref name="value"/> is less than zero.
+        /// </exception>
+        /// <since_tizen> 10 </since_tizen>
+        public Rectangle? Roi
+        {
+            get
+            {
+                return _roi;
+            }
+            set
+            {
+                if (value != null)
```

Comment:
impossible to set it back to null once it is not null?

Reply:
Actually, RoiTrackingConfiguration.Roi is mandatory for TrackAsync method.
So I'll change it to `public Rectangle Roi`, not nullable.